assignment6/src/Movement.java

```diff
@@ -0,0 +1,23 @@
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Map.Entry;
+import java.util.logging.Logger;
+
+public class Movement {
+    private final static Logger logger = Logger.getLogger(Movement.class.getName());
+
+    private int from;
+    private int to;
+
+    public Movement(int from, int to) {
+        this.from = from;
+        this.to = to;
+    }
+
+    public void print() {
+        logger.info("move from " + from + " to " + to);
+    }
+
+    public Entry<Integer, Integer> getValues() {
```

Comment:
I think converting to an Entry is a bad design. A move really isn't a key-value pair, it's swapping of a set of indices. All you're using Entry for is as a way of returning a pair of values - and you could just get the same effect by having accessors "getFrom" and "getTo".